enterprise/frontend/src/embedding-sdk/test/mocks/config.ts

```diff
@@ -4,8 +4,8 @@ export const createMockConfig = ({
   jwtProviderUri,
   ...opts
 }: {
-  jwtProviderUri: SDKConfig["jwtProviderUri"];
-} & Partial<SDKConfig>): SDKConfig => ({
+  jwtProviderUri: string;
+} & Partial<Omit<SDKConfig, "jwtProviderUri" | "apiKey">>): SDKConfig => ({
```

Comment:
I've tried to make this type and all its dependencies work, but it seems this isn't easy. I feel like this type is overly complicated. I don't have a definitive answer, but it seems when we introduce `apiKey` back. Some types in unit tests break since we used to assume `jwtProviderUri` would always be a string in some places. Like here the type of `jwtProviderUri` is `string`, when it could be optional since the config could be for an API key which doesn't contain `jwtProviderUri` at all.

Reply:
I agree. Maybe we can use `SdkConfigWithJwt` as the main one for tests?